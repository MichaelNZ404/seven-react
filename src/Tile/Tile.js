import React, { Component } from 'react';
import styled from 'styled-components';
import LogoImage from './logo.png';
import TileImage from './tile.jpg';

const Title = styled.span`
  font-size: 1.5em;
  text-align: center;
  color: #FFF;
`;

const LogoImageWrapper = styled.img`
  height: 25px;
  margin-right: 5px;
`;

const TileImageWrapper = styled.img`
  max-width: 100%;
`;

const BannerContainer = styled.div`
  position: relative;
  top: -34px;
  background-color: rgba(0, 0, 0, 0.7);
`;

const TileContainer = styled.div`
  max-height: 350px;
  display: inline-block;
`;

export class Banner extends React.Component {
  render() {
    return(
      <div>
        <BannerContainer>
          <LogoImageWrapper src={`${LogoImage}`} />
          <Title className='show-name'>{this.props.showName}</Title>;
        </BannerContainer>
      </div>
    );
  }
}

export class Tile extends React.Component {
    render() {
      return(
        <TileContainer>
          <TileImageWrapper src={`${TileImage}`} />
          <Banner showName="Home and Away" />
        </TileContainer>
      );
    }
  }
