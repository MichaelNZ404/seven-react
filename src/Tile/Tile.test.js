import React from 'react';
import { shallow, mount, render } from 'enzyme';

import {Banner, Tile} from './Tile.js'

test('Banner Matches snapshot', () => {
    const wrapper = render(<Banner showName="test" />);
    expect(wrapper).toMatchSnapshot();
});

test('Tile Matches snapshot', () => {
    const wrapper = render(<Tile />);
    expect(wrapper).toMatchSnapshot();
});

test('Banner title has correct length', () => {
    const wrapper = render(<Banner showName="test" />);
    expect(wrapper.find('.show-name').text()).toHaveLength(4);
});