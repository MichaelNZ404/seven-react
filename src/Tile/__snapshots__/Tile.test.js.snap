// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner Displays correct title 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bxivhb vgWHZ",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH fwSjfy",
              "src": "logo.png",
            },
            "children": Array [],
            "name": "img",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "data": ";",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "text",
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "src": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "src": undefined,
            },
          },
          Object {
            "attribs": Object {
              "class": "show-name sc-bdVaJa emICmq",
            },
            "children": Array [
              Object {
                "data": "test",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "data": ";",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "text",
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "sc-bwzfXH fwSjfy",
                "src": "logo.png",
              },
              "children": Array [],
              "name": "img",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "src": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "src": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          Object {
            "data": ";",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "sc-bwzfXH fwSjfy",
                  "src": "logo.png",
                },
                "children": Array [],
                "name": "img",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "src": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "src": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "type": "text",
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Banner MAtches snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bxivhb vgWHZ",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH fwSjfy",
              "src": "logo.png",
            },
            "children": Array [],
            "name": "img",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "data": ";",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "text",
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "src": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "src": undefined,
            },
          },
          Object {
            "attribs": Object {
              "class": "show-name sc-bdVaJa emICmq",
            },
            "children": Array [
              Object {
                "data": "test",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "data": ";",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "text",
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "sc-bwzfXH fwSjfy",
                "src": "logo.png",
              },
              "children": Array [],
              "name": "img",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "src": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "src": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          Object {
            "data": ";",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "sc-bwzfXH fwSjfy",
                  "src": "logo.png",
                },
                "children": Array [],
                "name": "img",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "src": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "src": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "type": "text",
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Banner Matches snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {},
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bxivhb vgWHZ",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH fwSjfy",
              "src": "logo.png",
            },
            "children": Array [],
            "name": "img",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "data": ";",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "text",
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "src": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "src": undefined,
            },
          },
          Object {
            "attribs": Object {
              "class": "show-name sc-bdVaJa emICmq",
            },
            "children": Array [
              Object {
                "data": "test",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "data": ";",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "text",
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "sc-bwzfXH fwSjfy",
                "src": "logo.png",
              },
              "children": Array [],
              "name": "img",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
                "src": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
                "src": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          Object {
            "data": ";",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "show-name sc-bdVaJa emICmq",
              },
              "children": Array [
                Object {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "sc-bwzfXH fwSjfy",
                  "src": "logo.png",
                },
                "children": Array [],
                "name": "img",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "src": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "src": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "type": "text",
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Tile Matches snapshot 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "sc-ifAKCX dtcItV",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-htpNat RlOql",
          "src": "tile.jpg",
        },
        "children": Array [],
        "name": "img",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {},
          "children": Array [
            Object {
              "attribs": Object {
                "class": "sc-bxivhb vgWHZ",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "sc-bwzfXH fwSjfy",
                    "src": "logo.png",
                  },
                  "children": Array [],
                  "name": "img",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "attribs": Object {
                      "class": "show-name sc-bdVaJa emICmq",
                    },
                    "children": Array [
                      Object {
                        "data": "Home and Away",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": Object {
                      "data": ";",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "text",
                    },
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                    "src": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                    "src": undefined,
                  },
                },
                Object {
                  "attribs": Object {
                    "class": "show-name sc-bdVaJa emICmq",
                  },
                  "children": Array [
                    Object {
                      "data": "Home and Away",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": ";",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "sc-bwzfXH fwSjfy",
                      "src": "logo.png",
                    },
                    "children": Array [],
                    "name": "img",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                      "src": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                      "src": undefined,
                    },
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                Object {
                  "data": ";",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "show-name sc-bdVaJa emICmq",
                    },
                    "children": Array [
                      Object {
                        "data": "Home and Away",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": Object {
                      "attribs": Object {
                        "class": "sc-bwzfXH fwSjfy",
                        "src": "logo.png",
                      },
                      "children": Array [],
                      "name": "img",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                        "src": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                        "src": undefined,
                      },
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "type": "text",
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "src": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "src": undefined,
        },
      },
      Object {
        "attribs": Object {},
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bxivhb vgWHZ",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "sc-bwzfXH fwSjfy",
                  "src": "logo.png",
                },
                "children": Array [],
                "name": "img",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "class": "show-name sc-bdVaJa emICmq",
                  },
                  "children": Array [
                    Object {
                      "data": "Home and Away",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": ";",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                  "src": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                  "src": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "class": "show-name sc-bdVaJa emICmq",
                },
                "children": Array [
                  Object {
                    "data": "Home and Away",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "data": ";",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "text",
                },
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "sc-bwzfXH fwSjfy",
                    "src": "logo.png",
                  },
                  "children": Array [],
                  "name": "img",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                    "src": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                    "src": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "data": ";",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "show-name sc-bdVaJa emICmq",
                  },
                  "children": Array [
                    Object {
                      "data": "Home and Away",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "sc-bwzfXH fwSjfy",
                      "src": "logo.png",
                    },
                    "children": Array [],
                    "name": "img",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                      "src": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                      "src": undefined,
                    },
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "text",
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "sc-htpNat RlOql",
            "src": "tile.jpg",
          },
          "children": Array [],
          "name": "img",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
            "src": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
            "src": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
